src/site/content/en/blog/prefers-color-scheme/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Hello Darkness, My Old Friend—Dark Theme is Here
+subhead:
+authors:
+  - thomassteiner
+date: 2019-05-23
+hero: macosx.png
+alt: macOS X Dark Mode preferences.
+description: |
+
+tags:
```

Comment:
```suggestion
tags:
  - post
```

post tag is required to show up on the blog landing page. I need to figure out a better way to make sure all blog posts get it automatically :\